lib/sinon/util/fake_timers.js

```diff
@@ -2,17 +2,34 @@
 
 var llx = require("lolex");
 
-exports.useFakeTimers = function () {
-    var now;
-    var methods = Array.prototype.slice.call(arguments);
+/**
+ *
+ * @param config {number|Date|Object} the unix epoch value to install with (default 0) or
+ */
+exports.useFakeTimers = function (args) {
```

Comment:
The sinonjs website uses GitHub Flavoured Markdown, so anything you can do on GitHub, you can do in the docs :)